### PR TITLE
update the geoResource's label asynchronously

### LINKED ISSUE #772: imported layer is not modifyable
When a layer is imported from local vector data, it cannot be modified (visibility, opacity).
Also feature highlighting is broken.

### PR DESCRIPTION
**Pull request by taulinger**
fixes #772

### CODE REVIEW

File: src/modules/map/components/olMap/services/VectorLayerService.js [hunk 1]
```diff
@@ -150,10 +150,14 @@ export class VectorLayerService {
 		const format = mapVectorSourceTypeToFormat(geoResource.sourceType);
 		const features = format.readFeatures(data);
 
-		// If we know a better name for the geoResource now, we update the label
+		/**
+		 * If we know a better name for the geoResource now, we update the the geoResource's label.
```
Comment (thiloSchlemmer) [inline src/modules/map/components/olMap/services/VectorLayerService.js:154]: typo 'the the'

**Review by thiloSchlemmer — APPROVED** [review]
Thx!

File: src/modules/map/components/olMap/services/VectorLayerService.js (hunk 1 above)
Comment (taulinger) [reply src/modules/map/components/olMap/services/VectorLayerService.js:154]: Thx

**Review by thiloSchlemmer — APPROVED** [review]
Thx

**Pull request merged**